TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -331,8 +335,10 @@ Case cases[] = {
     Case("Test detach", test_detach),
     Case("Test multi call and time measure", test_multi_call_time),
     Case("Test multi ticker", test_multi_ticker),
+#if !defined(__ARM_FM)  //FastModels not support time drifting test
     Case("Test timers: 1x ticker", test_case_1x_ticker),
     Case("Test timers: 2x ticker", test_case_2x_ticker)
+#endif
```

Comment:
if test_case_2x_ticker is skipped from compiled time do we need runtime check on line 149 ? 

What limitations are there for ARM_FM for this 2 test cases ? I would like to avoid having this ifdef there

Reply:
Good catch. my bad forgot to remove line 149, which suggested by @cmonr.

The reason was stated in the PR description:
>FastModels targets are simulator running on the x86 hosts.
As the nature of non-RealTime x86 OS and FastModels, timing accuracy is not guaranteed
mbed time-drifting-tests will failing all the time on FastModel targets

my first version to skip the test was in `line 149`, now it been changed to like `line 338`